var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -131,8 +131,7 @@ def install(self, spec, prefix):
         # In those SuiteSparse versions calling "make install" in one go is
         # not possible, mainly because of GraphBLAS.  Thus compile first and
         # install in a second run.
-        if (self.spec.version >= Version('5.4.0') and
-            self.spec.version <= Version('5.6.0')):
+        if (self.spec.version >= Version('5.4.0')):
```

Comment:
Can we change it with the more idiomatic:
```python
if '@5.4.0:' in self.spec:
```
?

Reply:
Thanks for the hint and done :)